### PR TITLE
ce_dd.prg support $ make USE_LOWLEVEL_PEXEC=no

### PR DESCRIPTION
**Pull request by miniupnp**
So it is easier to test without editing Makefile, c files, etc.

### CODE REVIEW

File: ce_dd_prg/Makefile [hunk 1]
```diff
@@ -43,13 +43,14 @@ CSRC     = mutex.c xbra.c acsi.c scsi_falcon.c scsi_tt.c \
...
-           harddrive_lowlevel.s
+SSRC     = gemdos_asm.s bios_asm.s screenirq.s
 
 ifeq ($(USE_LOWELEVEL_PEXEC),yes)
     CSRC += gemdos_pexec_lowlevel.c
+    SSRC += harddrive_lowlevel.s
 else
     CSRC += gemdos_pexec.c
+    CFLAGS += -DMANUAL_PEXEC
 endif
```
Comment (atarijookie) [inline ce_dd_prg/Makefile:55]: Which Pexec() is default with these changes? lowlevel (using sectors), or manual? Would that be the LOWLEVEL, which is on the start of the makefile, right?

please note: you have a typo there - it's **LOWLEVEL**, not _LOWELEVEL_  ;)

Comment (miniupnp) [inline ce_dd_prg/Makefile:55]: I changed nothing in default compiling behaviour, only allowed to compile other version without having to edit a bunch of files.
Default is what is in Makefile :
 USE_LOWELEVEL_PEXEC = yes

Comment (atarijookie) [inline ce_dd_prg/Makefile:55]: OK, and please fix that typo...

**Comment by atarijookie** [comment]
Thanks...

**Pull request merged**